pkg/platform/packet/template.go

```diff
@@ -228,6 +228,25 @@ provider "aws" {
   # "us-east-1": https://docs.aws.amazon.com/general/latest/gr/r53.html.
   region = "us-east-1"
 }
+
+{{- else if eq .Config.DNSProvider "cloudflare" }}
+module "dns" {
+  source = "../lokomotive-kubernetes/dns/cloudflare"
+
+  providers = {
+    cloudflare = cloudflare.default
+  }
+
+  cluster_name             = "{{ .Config.ClusterName }}"
+  controllers_public_ipv4  = module.packet-{{.Config.ClusterName}}.controllers_public_ipv4
```

Comment:
Maybe put all those parameters in locals, so we don't have to repeat that much Go templating? Something like:
```hcl
locals {
  cluster_name = "{{ .Config.ClusterName }}"
  controllers_public_ipv4  = module.packet-{{.Config.ClusterName}}.controllers_public_ipv4
}

module "dns" {
  source = "../lokomotive-kubernetes/dns/cloudflare"

  cluster_name             = local.cluster_name
...
```

Reply:
Following the removal of the repeated `module "dns" {}` block, this is now irrelevant.